spyder/config/main.py

```diff
@@ -83,6 +83,8 @@
               'memory_usage/timeout': 2000,
               'cpu_usage/enable': False,
               'cpu_usage/timeout': 2000,
+              'clock/enable': True,
+              'clock/timeout': 2000,

```

Comment:
Perhaps this should be updated every minute? In other words, this value should be `60000` instead of `2000`?

Reply:
This depends on when in time the timer starts so the only "issue" here is that if we put a larger value (>2000) then there might be situations in which the clock on Spyder and the clock of the operating system (taskbar etc..) will be "off" by 1 minute. 

So I think we should probably have 1000 to avoid this issue.

Thoughts @ccordoba12 ?

Reply:
> So I think we should probably have 1000 to avoid this issue.

Ok, agreed.